FirebaseStorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 8.0.1
```

Comment:
We only do patches for emergency breakages now. Otherwise all Firebase products version together with minor updates.  This should be 8.1.0 or if it's serious enough we should cherry-pick to 8.0.0 which hasn't released yet.